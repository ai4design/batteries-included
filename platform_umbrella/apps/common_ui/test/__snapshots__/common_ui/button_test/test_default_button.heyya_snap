<button class="pc-button pc-button--light-shadow pc-button--md ">
  
  
    
  

  
  Test Button


</button>