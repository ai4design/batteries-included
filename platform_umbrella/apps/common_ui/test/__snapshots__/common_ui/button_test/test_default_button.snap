"<button class=\"relative inline-flex items-center justify-center p-0.5 mb-2 mr-2 rounded-lg overflow-hidden text-sm font-medium text-gray-900 transition-all ease-in-out duration-300 hover:scale-110 bg-gradient-to-tl from-primary-500 to-secondary-500 group/button focus:ring-4 focus:outline-none focus:ring-pink-300 hover:bg-gradient-to-bl hover:from-secondary-500 hover:to-primary-500 hover:text-white hover:shadow-lg hover:shadow-secondary-500/20\">\n  <span class=\"relative px-5 py-2.5 w-full rounded-md transition-all ease-in-out duration-300 bg-white group-hover/button:bg-opacity-0\">\n    Test Button\n  </span>\n</button>"